sql/parser/scan.go

```diff
@@ -152,6 +152,17 @@ func (s *scanner) scan(lval *sqlSymType) {
 		if isDigit(s.peek()) {
 			s.scanParam(lval)
 			return
+		} else if s.syntax == Modern {
+			// TODO(pmattis): This should really be prefixed with '@', but that
```

Comment:
So I assume that means we're going to change what we're doing for index indirection?